app/routes.py

```diff
@@ -1,2 +1,50 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+class Planet:
+    def __init__ (self, id, name, description, moon_count):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.moon_count = moon_count
+    
+    def to_json(self):
+        return {
+            "id": self.id, 
+            "name": self.name,
+            "description": self.description, 
+            "moon_count": self.moon_count
+        }
+
+planets = [
+    Planet(1, "Mercury", "red planet", 1),
+    Planet(2, "Venus", "blue planet", 3),
+    Planet(1, "Earth", "water planet", 1)
+]
+
+def validate_planet(id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message": f"Planet {id} is not valid"}, 400))
+
+    for planet in planets:
+        if planet.id == id:
+            return planet
+    
+    return abort(make_response({"message": f"Planet {id} not found"}, 404))
+
+@planets_bp.route("", methods=["GET"])
+def get_planets():
```

Comment:
Nice job naming the method to describe what this route does.

In the future, when you split your routes up into distinct route files (cat_routes.py and dog_routes.py for example in a routes directory) then you could have method names like get_all() and get_one() since we know that all the routes in cat_routes.py are related to the Cat class.